charts/akeyless-secure-remote-access/Chart.yaml

```
@@ -14,14 +14,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 
-version: 0.28.5
+version: 0.29.0
 
 
-appVersion: v1.0.0_0.16.1_1.4.2
+appVersion: 2.0.0_1.4.2
```

```suggestion
appVersion: 1.0.0_1.4.2
```

version will be bump to 2 after release